.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+python:
+  - "2.7"
+sudo: required
+services:
+  - docker
+env:
+  global:
+    - PA_TEST_ONLINE_INSTALLER=true
+  matrix:
+    - OTHER_TESTS=true
+    - PRODUCT_TEST_GROUP=0
+    - PRODUCT_TEST_GROUP=1
+    - PRODUCT_TEST_GROUP=2
+    - PRODUCT_TEST_GROUP=3
+    - PRODUCT_TEST_GROUP=4
+    - PRODUCT_TEST_GROUP=5
+    - PRODUCT_TEST_GROUP=6
+    - PRODUCT_TEST_GROUP=7
+install: 
+  - pip install --upgrade pip==6.1.1
+  - pip install -r requirements.txt
+script: 
+  - |
+    if [ -v PRODUCT_TEST_GROUP ]; then
+      PRODUCT_TESTS=$(find tests/product/ -name '*py' | grep -v __init__ | xargs wc -l | sort -n | head -n -1 | awk '{print $2}' | awk "NR % 8 == $PRODUCT_TEST_GROUP" | tr '\n' ' ')
```

Comment:
Suggest combining the last two awk commands into awk "NR % 8 == $PRODUCT_TEST_GROUP { print \$2 }", but it's up to you.